### ISSUE TITLE
allow shuffling of sequences in provided dataloader (dlomix dataset class)

### ISSUE THREAD

**Issue by ayla-s** [wilhelm-lab/dlomix#70]
When using the dlomix dataset class, the returned dataloader object, when accessing the class properties **tensor_<train|val|test>_data** (see example below), currently contains the sequences per batch in a **sorted / not shuffled order**. Atm, there seems to be no way to activate data shuffling when using the dlomix dataset.
However, data shuffling is an existing feature of the internally used PyTorch DataLoader and its tf equivalent.

Having a parameter that allows the user to switch on shuffling would be super helpful. The default value of the parameter could still be False for backward compatibility.

current usage example (PyTorch):
```
data = ChargeStateDataset(**kwargs)
for batch in data.tensor_train_data:
            train_seq = batch[seq_col]
            train_label = batch[charge_col]
```

suggested integration:
```
data = ChargeStateDataset(**kwargs, shuffling=True)      # new parameter added
for batch in data.tensor_train_data:
            train_seq = batch[seq_col]
            train_label = batch[charge_col]
```

Any other integration of a parameter allowing to switch on shuffling would also be fine :)

Thank you! ☺️
